prettier.el

```diff
@@ -573,6 +565,20 @@ IDENTIFICATION and CONNECTED have the same meaning as
               identification
               connected)))
 
+(defun prettier-el-version ()
```

Comment:
Thanks for turning it into a function, looks great.

I've been trying to follow the [coding conventions](https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html), which recommend to
> [u]se two hyphens to separate prefix and name if the symbol is not meant to be used by other packages. 

Not a big deal, just wanted to make sure you're aware of this convention. I'm leaning towards considering it a package-private function but can see an argument to be made the other way.

Reply:
Fixed and rebased.